homework1/index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Simple Art</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Baloo+2:wght@400;600&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/modern-normalize/1.1.0/modern-normalize.min.css"
+    />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <body>
+      <div class="burger" id="burger">
+        <div class="bar1"></div>
+        <div class="bar2"></div>
+        <div class="bar3"></div>
+      </div>
+      <header class="header">
+        <div class="container header-container">
+            <a href="#" class="logo-link link" lang="en">
+              <img src="./img/logo-SA.svg" alt="logo">
+            </a>
+            <nav class="nav-container sidebar hidden" id="sidebar">
+              <ul class="list site-nav">
+                <li><a href="" class="nav-item link">Courses</a></li>
+                <li>
+                  <a href="" class="nav-item link">Teachers</a>
+                </li>
+                <li><a href="" class="nav-item link">Prices</a></li>
+                <li><a href="" class="nav-item link">About us</a></li>
+              </ul>
+              <button type="submit" class="button">
+                Apply now
+              </button>
+          </nav>
+      </header>
+
+      <main>
+        <section class="hero-section">
+          <div class="container">
+            <div class="hero-container">
+              <h1 class="hero-header">Desperate to learn drawing? We’ll teach you!</h1>
+              <p class="hero-description">Our school has more than 20 years of experience in teaching people who always thought that they don’t have even a slightest hint of a talent. Some other descriptive stuff that I’m too lazy to make up.</p>
+              <p class="hero-list-title">With us you will get:</p>
+              <ul class="hero-list">
+                <li class="hero-list-item">Lorem ipsum dolor sit amet, consectetur adipiscing elit. </li>
+                <li class="hero-list-item">Vestibulum commodo id amet non facilisis in.</li>
+                <li class="hero-list-item">Est eros, vestibulum cursus luctus luctus aenean molestie. </li>
+                <li class="hero-list-item">Augue ultricies non tristique malesuada justo neque mi, pellentesque lacus.</li>
+              </ul>
+              <button class="button" data-modal-open type="button">
+                Apply now
+              </button>
+            </div>
+            <div class="img-container">
```

Comment:
when i change my display width the image is not on full height
![image](https://user-images.githubusercontent.com/28695044/198216873-dea6029c-15eb-4085-bac5-b8298255bb94.png)